### PR TITLE
Sound changes

### PR DESCRIPTION
**Pull request by MEEPofFaith**
Changes various sounds, adds one new sound which I downloaded and edited slightly.
[Original Sound](https://www.zapsplat.com/music/science-fiction-short-energy-pulse-explosion-high-pitch-synth-futuristic/)

### CODE REVIEW

**Review by MEEPofFaith — COMMENTED** [review]
Point**Defenct**Turret

**Comment by GlennFolker** [comment]
ach nein

**Comment by Syncinus** [comment]
These are decent, but I feel like there should be a bit more variety, with the number of units and stuff added, I feel like just the current amount of available sounds don't fit everything.

**Comment by sk7725** [comment]
Please *add* new sounds instead of replacing them, we need WAY more sounds hhhhhh
especially the ammoless \*click\*

**Comment by MEEPofFaith** [comment]
I'll undo the remove duplicate sounds thing later because apparently they're different, thoughI can't tell the difference.

**Comment by MEEPofFaith** [comment]
"rauilgun" My spelling is on point

**Comment by MEEPofFaith** [comment]
[So yea, here's a video that kinda showcases the changes to sounds I made kinda not really. (You would need to know how they normally sound like to really tell the difference.)](https://youtu.be/lhu36WvYmKM)

File: core/src/mindustry/content/UnitTypes.java [hunk 1]
```diff
@@ -520,7 +520,8 @@ public void load(){
             mechSideSway = 0.25f;
             range = 40f;
 
-            weapons.add(new Weapon(){{
+            weapons.add(
+            new Weapon(){{
```
Comment (summetdev) [inline core/src/mindustry/content/UnitTypes.java:524]: unnecessary new line here

File: core/src/mindustry/content/UnitTypes.java [hunk 2]
```diff
@@ -876,7 +880,8 @@ public void load(){
             engineOffset = 5.5f;
             range = 140f;
 
-            weapons.add(new Weapon(){{
+            weapons.add(
+            new Weapon(){{
```
Comment (summetdev) [inline core/src/mindustry/content/UnitTypes.java:884]: and here

File: core/src/mindustry/content/UnitTypes.java [hunk 3]
```diff
@@ -899,7 +904,8 @@ public void load(){
             faceTarget = false;
             armor = 4f;
 
-            weapons.add(new Weapon(){{
+            weapons.add(
+            new Weapon(){{
```
Comment (summetdev) [inline core/src/mindustry/content/UnitTypes.java:908]: and also here

File: core/src/mindustry/content/UnitTypes.java [hunk 4]
```diff
@@ -1270,7 +1278,8 @@ public void load(){
 
                     shootCone = 180f;
                     ejectEffect = Fx.none;
-                    shootSound = Sounds.none;
+                    hitSound = Sounds.explosionbig;
+
```
Comment (summetdev) [inline core/src/mindustry/content/UnitTypes.java:1282]: and also here again

File: core/src/mindustry/content/UnitTypes.java (hunk 1 above)
Comment (MEEPofFaith) [reply core/src/mindustry/content/UnitTypes.java:524]: All the other weapons are like this, so I made these 3 like the others; make formatting consistent.

File: core/src/mindustry/content/UnitTypes.java (hunk 2 above)
Comment (MEEPofFaith) [reply core/src/mindustry/content/UnitTypes.java:884]: All the other weapons are like this, so I made these 3 like the others; make formatting consistent.

File: core/src/mindustry/content/UnitTypes.java (hunk 3 above)
Comment (MEEPofFaith) [reply core/src/mindustry/content/UnitTypes.java:908]: All the other weapons are like this, so I made these 3 like the others; make formatting consistent.

Yes I did just make the same comment to 3 comments.

**Comment by Syncinus** [comment]
The railgun sound is just a deepened version of the shotgun sound, which I don't think really fits a railgun, it's sound should probably just be a really loud bang and then it gets quiter, kind of like when you shoot a meltdown at really low power and it then fires a very quick shot, which sounds kind of like that.

I think sounds that need to be added in general:
- Railgun sound (Omura, Foreshadow)
- Sapping laser sound (Spiroct, Arkyid)
- Continous laser sound like meltdown, that is a bit less insane (Vela)
- Charge and huge laser sound (Corvus)
- Bomb drop sound (Quad)
- Huge cannon sound kind of like meltdown (Toxopid)
- (Maybe) some kind of sound for healing guns (Poly, Mega)

And then I think some stuff that would fit is the salvo sound being used on sei.
I see that some of this has already been added in the PR (like quad has a new sound)

**Comment by sk7725** [comment]
Make the sap laser sound an adult male voice saying "sap"

**Comment by MEEPofFaith** [comment]
> Make the sap laser sound an adult male voice saying "sap"

As much as I want to make all the sounds my voice, I won't.

**Comment by MEEPofFaith** [comment]
> The railgun sound is just a deepened version of the shotgun sound, which I don't think really fits a railgun, it's sound should probably just be a really loud bang and then it gets quiter, kind of like when you shoot a meltdown at really low power and it then fires a very quick shot, which sounds kind of like that.

~~Hire Eye of Darkness to do that/steal from Advance Content.~~

**Comment by sk7725** [comment]
@EyeOfDarkness

**Comment by EyeOfDarkness** [comment]
AC's sounds is mostly just edited and mixed mindustry and cosmoteer sounds.

**Comment by GlennFolker** [comment]
spell his name correvtly ~~or i kill you~~

**Comment by MEEPofFaith** [comment]
I have no idea how to rename a commit and I just checked DMs and found I put the wrong name.

**Comment by HaphLife** [comment]
The game's sound design is pretty good, but I totally agree that it could use more variety. Both in sound effects, and background music. 👍

**Comment by MEEPofFaith** [comment]
Nothing much I can think of, so I guess I'll set this is Ready.

**Comment by MEEPofFaith** [comment]
Hm new sound doesn't exactly fit well.

**Comment by MEEPofFaith** [comment]
Anuke seems to be doing this himself, so I'll close this.

**Pull request closed without merging**